CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+
+## Unreleased as of Sprint 71 ending 2017-10-16
+
+### Added
+- Add missing usage types for integration [(#110)](https://github.com/ManageIQ/manageiq-consumption/pull/110)
+- Now rate create a zero tier after create [(#105)](https://github.com/ManageIQ/manageiq-consumption/pull/105)
+- Add US8 currency [(#104)](https://github.com/ManageIQ/manageiq-consumption/pull/104)
+- Changes that correspond with "v5" commit of "chargio" branch in ManageIQ repo [(#95)](https://github.com/ManageIQ/manageiq-consumption/pull/95)
```

Comment:
Changes that correspond with "v5" commit of "chargio" branch in ManageIQ repo [(#95)](https://github.com/ManageIQ/manageiq-consumption/pull/95). They allow the new chargeback to be plugged in subtituting the old rating